src/graph.js

```diff
@@ -60,6 +61,19 @@ module.exports = (function () {
         return this;
     };
 
+    Graph.prototype.dontRender = function() {
+        this._renderNow = false;
+
+        return this;
+    };
+
+    Graph.prototype.doRender = function() {
```

Comment:
> As is (i.e. disableAutoRender before the forEach, enableAutoRender after), or let the user decide?

I don't have strong opinions one way or the other. As long as running `addNodes`/`addEdges` doesn't change the end state of `_renderNow` (or whatever you choose to call that private variable). For example, `graph.disabledAutoRender(); graph.addNodes([..]);` should result in the auto render boolean still `false`.

Not worried about git commit history; do whatever you think is appropriate.